hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/utils/ConversionUtilsTest.java

```diff
@@ -16,9 +16,19 @@
 
 package com.hedera.node.app.service.contract.impl.test.utils;
```

Comment:
This unit test class is missing tests for several `ConversionUtils.java` new methods.  Everything else in this PR is at 100% ...